src/hooks/v2v3/contractReader/useHeldFeesOf.ts

```diff
@@ -1,25 +1,24 @@
 import { ProjectMetadataContext } from 'contexts/shared/ProjectMetadataContext'
-import { V2V3ContractsContext } from 'contexts/v2v3/Contracts/V2V3ContractsContext'
 import { useContext, useEffect, useState } from 'react'
 import { sumHeldFees } from 'utils/v2v3/math'
+import { useProjectPrimaryEthTerminal } from '../V2V3ProjectContracts/projectContractLoaders/useProjectPrimaryEthTerminal'
 
 export function useHeldFeesOf() {
   const [heldFees, setHeldFees] = useState<number>()
 
   const { projectId } = useContext(ProjectMetadataContext)
-  const { contracts } = useContext(V2V3ContractsContext)
+  if (!projectId) return
+  const { JBETHPaymentTerminal } = useProjectPrimaryEthTerminal({ projectId })
```

Comment:
**suggestion** We should just refactor this hook to use `useContractReader` like normal.

Reply:
dw bout it for now tho @filipviz.

But quick question: whats the reason behind the changes here?

Reply:
Oh. @filipviz i see! It wasn't using the projects terminal.

In this case, we should already have the primary terminal in context.

Try getting `JBETHPaymentTerminal` from `contracts` here:
```
  const { contracts } = useContext(V2V3ProjectContractsContext)
```

Also, will need to remove line 10: `if (!projectId) return`. React doesn't like hooks running conditionally 🙏

Reply:
> But quick question: whats the reason behind the changes here?

The old version just got the `JBETHPaymentTerminal`, so if a project was using 3_1, 3_1_1, etc. it wouldn't work

Reply:
Ok, will leave for now! If we're leaving, this should be good now.

Reply:
> Oh. @filipviz i see! It wasn't using the projects terminal.
> 
> In this case, we should already have the primary terminal in context.
> 
> Try getting `JBETHPaymentTerminal` from `contracts` here:
> 
> ```
>   const { contracts } = useContext(V2V3ProjectContractsContext)
> ```
> 
> Also, will need to remove line 10: `if (!projectId) return`. React doesn't like hooks running conditionally 🙏

Oh didn't see this. Ok will do – sorry for some of this. Not too familiar with the contract abstractions on JBM